pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarMetadata.java

```diff
@@ -314,15 +367,42 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName schemaTableName,
         builder.addAll(getColumns(null, schema, new HashSet<>(), new Stack<>(), new Stack<>()));
 
         if (withInternalColumns) {
-            PulsarInternalColumn.getInternalFields().stream().forEach(new Consumer<PulsarInternalColumn>() {
-                @Override
-                public void accept(PulsarInternalColumn pulsarInternalColumn) {
-                    builder.add(pulsarInternalColumn.getColumnMetadata(false));
-                }
-            });
+            PulsarInternalColumn.getInternalFields()
+                    .stream()
+                    .forEach(pulsarInternalColumn -> builder.add(pulsarInternalColumn.getColumnMetadata(false)));
+        }
+         return builder.build();
+    }
+    @VisibleForTesting
+    static Type convertPulsarType(SchemaType pulsarType) {
+        switch (pulsarType) {
+            case BOOLEAN:
+                return BooleanType.BOOLEAN;
+            case INT8:
+                return TinyintType.TINYINT;
+            case INT16:
+                return SmallintType.SMALLINT;
+            case INT32:
+                return IntegerType.INTEGER;
+            case INT64:
+                return BigintType.BIGINT;
+            case FLOAT:
+                return RealType.REAL;
+            case DOUBLE:
+                return DoubleType.DOUBLE;
+            case NONE:
+            case BYTES:
+                return VarbinaryType.VARBINARY;
+            case STRING:
+                return VarcharType.VARCHAR;
+            case DATE:
```

Comment:
We should use the corresponding Presto time data types here:

https://prestodb.github.io/docs/current/language/types.html